libs/dh/shared/test-util-metering-point/src/lib/dh-transloco-testing.module.ts

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
suggestion: move this to a new library under `dh/shared`. Currently we only have metering point domain but this testing module will be used across all domains.

Reply:
I've will move it, however I'm pretty sure that we all talked about ealier that we should'nt lift things up before the usecase appears, and properly other domains are not upcomming in the near future :)

Reply:
You have valid points, @frontend-specialisten, but i18n is always a cross-cutting concern so @dzhavat also has a strong case.